### ISSUE TITLE
Richland Parish (Louisiana) Assessor’s Office

### ISSUE THREAD

**Issue by 1ec5** [osmlab/editor-layer-index#1458]
The tax assessor’s office of Richland Parish, Louisiana, has released 6-inch imagery of the entire parish and 3-inch imagery of its towns and is distributing them through Louisiana State University.

## URL of the imagery
https://tiles.arcgis.com/tiles/SDQDNhpG8jikA0D1/arcgis/rest/services/Richland_Parish_6_inch_2016/MapServer
https://tiles.arcgis.com/tiles/SDQDNhpG8jikA0D1/arcgis/rest/services/Richland_Parish_3_inch_imagery_2017/MapServer

## Two letter country code
US

## Desired layer name
RPAO_2016
RPAO_2017

## License
Creative Commons Attribution-ShareAlike: [2016](https://atlas.ga.lsu.edu/datasets/rpao2016/), [2017](https://atlas.ga.lsu.edu/datasets/rpao2017/)

## Is the license compatible with OSM?
TBD – need to [obtain a waiver](https://github.com/osmlab/editor-layer-index/blob/gh-pages/FAQ.md#where-can-i-find-an-imagery-license-waiver) first

## Attribution (text, URL, required?)
[Richland Parish Assessor’s Office](http://www.rpao.org/). Distributed by [Atlas](http://atlas.lsu.edu), LSU Department of Geography and Anthropology.

## Privacy policy (URL)
https://lsu.edu/privacy/

## Icon
http://www.rpao.org/favicon.ico

## Bounding polygon
The [2016 imagery](https://maps.ga.lsu.edu/rpao2016/) corresponds to the [Richland Parish](https://www.openstreetmap.org/relation/1838159) boundary.
The [2017 imagery](https://maps.ga.lsu.edu/rpao2017/) corresponds to simple bboxes around the towns of [Delhi](https://www.openstreetmap.org/relation/132164), [Mangham](https://www.openstreetmap.org/relation/132165), [Rayville](https://www.openstreetmap.org/relation/132166), and [Start](https://www.openstreetmap.org/node/151602974) in Richland Parish.
The interactive notebook in https://github.com/osmlab/editor-layer-index/pull/1431#issuecomment-1026146281 can help to generate the GeoJSON for these layers.

**Comment by cicku** [comment]
How about getting a waiver before submitting here... 🤣

**Comment by 1ec5** [comment]
Fair enough, I just needed a convenient place to put these notes when sharing it to locals so they can ask for permission, but I can use a Gist next time. Sorry for the noise!

**Comment by rbuffat** [comment]
@1ec5 it is perfectly fine to create the issue without having already a waiver. Issues are places where discussion can and should happen.
